build.sh

```diff
@@ -27,7 +27,7 @@ PREFIX_H="$PREFIX_BUILD/include/"
 
 PREFIX_ROOTFS="$PREFIX_FS/root/"
 
-CROSS=arm-phoenix-
+CROSS=i386-pc-phoenix-
```

Comment:
Isn't this script supposed to be generic? Why not define it in build.project in core script?

Reply:
I removed the CROSS variable from build.sh. It should be defined inside build.project file.